### PR TITLE
Update Getting Started with more granular options

### PR DESCRIPTION
**Pull request by annezazu**
## Description

This is just a quick change to allow for an easier time for folks joining this work with an emphasis on getting folks connected to both other contributors and other resources. It should help streamline getting information folks need to properly get started and connected in the project at a glance. 

## Types of changes

Enhancement 

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal). 
- [ ] I've updated related schemas if appropriate.

### CODE REVIEW

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Getting Started
 
-Welcome! Let's get started building with blocks. Blocks are at the core of extending WordPress. You can create custom blocks, your own block patterns, or combine them together to build a block theme.
+Welcome! Let's get started building with blocks. Blocks are at the core of extending WordPress. You can create custom blocks, your own block patterns, or combine them together to build a block theme. At a high level, here are a few ways to begin your journey but read on to explore more:
+
+- Join the [slack community](https://make.wordpress.org/chat/) to join meetings, ongoing conversations, and more.
```
Comment (priethor) [inline docs/getting-started/README.md]:
```suggestion
- Join the [Slack community](https://make.wordpress.org/chat/) to join meetings, ongoing conversations, and more. 
```

**Review by priethor — APPROVED** [review]
Love to see this; it is indeed a great summary on how to get started ❤️ 

I'm thinking it might be better to move Slack a bit down, as it can be a bit overwhelming at first: you land on different channels with ongoing hard-to-follow discussions. While it is true newcomers can just ask, it is likely that they just get redirected to GitHub or other resources 🤔

Also, ideally Make Core would be listed after `developer.wordpress.org`, as the latter contains less meta content, but unfortunately, right now most developer-oriented content is published in Make Core.

Nonetheless, I'm happy to merge and iterate later.

**Comment by annezazu** [comment]
>I'm thinking it might be better to move Slack a bit down, as it can be a bit overwhelming at first: you land on different channels with ongoing hard-to-follow discussions. While it is true newcomers can just ask, it is likely that they just get redirected to GitHub or other resources

I had the opposite feeling :D I wanted folks to get into slack to increase the chances of them finding community (which helps folks stick around). 

>Also, ideally Make Core would be listed after developer.wordpress.org, as the latter contains less meta content, but unfortunately, right now most developer-oriented content is published in Make Core.

In light of both things, I'll change around the order around with the roadmap first. It's always helpful to know, at a glance, where we're headed.

**Comment by priethor** [comment]
> I had the opposite feeling :D

I also agree with that point of view; that's why I don't feel too strongly either way 😅 . I have seen both: folks joining during a core-editor chat and feeling the community aspect from the start, and others asking things and not getting a clear answer.

We don't have direct feedback on either direction, do we?

**Comment by annezazu** [comment]
I don't feel strongly either - let's ship and iterate as we need :)

**Pull request merged**